src/results/PersonalBenefits.js

```diff
@@ -88,10 +88,10 @@ class PersonalBenefits extends React.Component<Props> {
                   </span>{" "}
                     € / {benefit.periode}
                   </Grid>
-                  <Grid item xs={6} sm={3}>
+                  <Grid className={"Separator"}item xs={6} sm={3}>
                     <Link to={benefit.url}>
                       <button
-                          style={{float: "right"}}
+                          style={{float: "right", marginTop: 17}}
```

Comment:
Are you sure you want to embed style in the element?

Reply:
It got already a class so I don't know really.